<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Konf</class>
 <widget class="QDialog" name="Konf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Konfigurationsmodus</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="1" column="1" rowspan="2">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Bestimmung der Höhe</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="4" column="1">
         <widget class="QDoubleSpinBox" name="dSpinBx_z_2">
          <property name="suffix">
           <string>cm</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-1000000000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>0.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QDoubleSpinBox" name="dSpinBx_calcHigh">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string>cm</string>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Z-Koordinate (max 0m)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" rowspan="2">
         <widget class="QDoubleSpinBox" name="dSpinBx_y_2">
          <property name="suffix">
           <string>cm</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-1000000000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Beta:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="Btn_Hchange">
          <property name="text">
           <string>übernehmen</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <pointsize>24</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Höhe =</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="lb_beta">
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="2">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>X-Koordinate</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Y-Koordinate</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="dSpinBx_x_2">
          <property name="suffix">
           <string>cm</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-1000000000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Bestimmung des Radius</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Z-Koordinate (max 0m)</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QDoubleSpinBox" name="dSpinBx_calcRadius">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string>cm</string>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="lb_alpha">
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="dSpinBx_y">
          <property name="suffix">
           <string>cm</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-1000000000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="0" rowspan="2">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>24</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Radius = </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>X-Koordinate</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="Btn_Rchange">
          <property name="text">
           <string>übernehmen</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1" rowspan="2">
         <widget class="QDoubleSpinBox" name="dSpinBx_z">
          <property name="suffix">
           <string>cm</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-1000000000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>0.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Alpha:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Y-Koordinate</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="dSpinBx_x">
          <property name="suffix">
           <string>cm</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-1000000000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Bestimmung der Geschwindigkeit</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Federspann-&lt;br/&gt;energie&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="dSpinBx_energie">
          <property name="prefix">
           <string/>
          </property>
          <property name="suffix">
           <string>J</string>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="dSpinBx_calcV">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string>cm/s</string>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="Btn_Vchange">
          <property name="text">
           <string>übernehmen</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Masse</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="dSpinBx_mass">
          <property name="suffix">
           <string>g</string>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_16">
          <property name="font">
           <font>
            <pointsize>24</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>v = </string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Moskito-Maße</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="dspinBx_high">
          <property name="suffix">
           <string>cm</string>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="dSpinBx_v">
          <property name="suffix">
           <string>cm/s</string>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Radius</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Höhe</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Geschwindigkeit&lt;br/&gt;des Geschlosses&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="dspinBx_radius">
          <property name="suffix">
           <string>cm</string>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Verstecken</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dSpinBx_x</tabstop>
  <tabstop>dSpinBx_y</tabstop>
  <tabstop>dSpinBx_z</tabstop>
  <tabstop>dSpinBx_calcRadius</tabstop>
  <tabstop>Btn_Rchange</tabstop>
  <tabstop>dSpinBx_x_2</tabstop>
  <tabstop>dSpinBx_y_2</tabstop>
  <tabstop>dSpinBx_z_2</tabstop>
  <tabstop>dSpinBx_calcHigh</tabstop>
  <tabstop>Btn_Hchange</tabstop>
  <tabstop>dSpinBx_energie</tabstop>
  <tabstop>dSpinBx_mass</tabstop>
  <tabstop>Btn_Vchange</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Konf</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>727</x>
     <y>519</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dSpinBx_x</sender>
   <signal>valueChanged(double)</signal>
   <receiver>dSpinBx_x_2</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>718</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dSpinBx_y</sender>
   <signal>valueChanged(double)</signal>
   <receiver>dSpinBx_y_2</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>718</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dSpinBx_x_2</sender>
   <signal>valueChanged(double)</signal>
   <receiver>dSpinBx_x</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>718</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dSpinBx_y_2</sender>
   <signal>valueChanged(double)</signal>
   <receiver>dSpinBx_y</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>718</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dSpinBx_z</sender>
   <signal>valueChanged(double)</signal>
   <receiver>dSpinBx_z_2</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>718</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dSpinBx_z_2</sender>
   <signal>valueChanged(double)</signal>
   <receiver>dSpinBx_z</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>718</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
